minifi/minifi-nar-bundles/minifi-framework-bundle/minifi-framework/minifi-resources/src/main/resources/conf/bootstrap.conf

```diff
@@ -134,8 +134,13 @@ java.arg.14=-Djava.awt.headless=true
 # Enabling C2 Uncomment each of the following options
 #c2.enable=true
 ## define protocol parameters
+# legacy properties
 #c2.rest.url=
 #c2.rest.url.ack=
+# new c2 properties
+#c2.rest.api=
```

Comment:
Naming is very subjective (as always) just trying to provide alternates maybe they are better or owrse will leave it to you or anyone else who reviews it:
- c2.rest.host - maybe this is misleading as it could contain not just the host
- c2.rest.base or c2.rest.path.base

What do you think?

Reply:
I went with c2.rest.path.base to be consistent with the other properties